### ISSUE TITLE
Summary text assumes quartets

### ISSUE THREAD

**Issue by cameronc137** [JeffersonLab/japan#240]
It looks like the summary text assumes quartets and this is breaking our downstream good time, events, and charge measurements as well. 

Probably we can work around this in the downstream codes, but it should be fixed here too.

**Comment by cameronc137** [comment]
https://github.com/JeffersonLab/japan/blob/7580f72f485055e5a60b74e44d580a95bd748a8c/Analysis/src/QwPromptSummary.cc#L159

**Comment by rahmans1** [comment]
Yeah. It assumes quartets. Downside of hardcoding. Probably do this through a configuration file. I have an update to prompt summary in the works but it's stalled because of other priorities. Will try to get it in as soon as I possibly can.

**Comment by sanghwapark** [comment]
This needs to be fixed relatively soon even if it's again hard-coded fix so that we have our charge counter works properly for the shift crew. We could live with correcting the charge counter back, but we need a fix for this and probably re-prompt for the future.

**Comment by cameronc137** [comment]
To stop-gap hack this to work from the 240/octet time onwards (until we fix the prompt summary and respin) I've multiplied runs after 3874-> good charge by 2x in the acc_charge.py script and it shows +1.5 more Coulombs as expected.

**Comment by cameronc137** [comment]
This was worked around during PREX, the thing that cared was our python script for getting charge into the charge plotter, but we should probably fix this for Respins so we don't have wrong information floating around.

@rr521111

**Comment by sanghwapark** [comment]
it's the other way around.. charge script kept assuming quartets because the summary output didn't get updated. I agree, it would be good to get it fixed for respin.

**Comment by rahmans1** [comment]
I remember trying to solve this and then getting distracted with other stuff. I will take a look tomorrow at the branch I was working on.

**Comment by paulmking** [comment]
@rahmans1:  @rr521111 and I talked last night about what ought to be a quick solution, so check in with him about it.

**Comment by cameronc137** [comment]
Has there been progress on this?

**Comment by paulmking** [comment]
I thought we had it fixed in develop before I merged back into operations.
What is the current behavior?

Sent from my mobile device.

________________________________

**Comment by cameronc137** [comment]
Yes, I think it’s fine. I was seeing an issue with pvdb, but it was a too-aggressive cut from Sanghwa’s script that did it.
